src/backend/gl/src/device.rs

```diff
@@ -285,9 +285,9 @@ impl d::Device<B> for Device {
                     if status != 0 {
                         if !log.is_empty() {
                             warn!("\tLog: {}", log);
+                        } else {
```

Comment:
this is incorrect: if status is success, there can still be error log that we want to print out the log, but this wouldn't mean an error. I believe the old code is correct. Would you mind posting your log and shader that you are trying to link?